llvm/test/tools/llvm-objcopy/ELF/compress-sections-within-segment.s

```diff
@@ -30,3 +35,24 @@ Sections:
     Type:     SHT_PROGBITS
     Flags:    [ SHF_ALLOC, SHF_COMPRESSED ]
```

Comment:
`SHF_ALLOC` here and below and the explicit `Address` and `Offset` parameters in the second case seem unnecessary? Similarly, the `VAddr` of the segment seems like it isn't required? In general, we should keep the YAML to a minimum to make it clear what is actually important to the behaviour.

Reply:
Removed addresses